python/simplesauce/session.py

```diff
@@ -0,0 +1,85 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.remote.remote_connection import RemoteConnection
+from simplesauce.options import SauceOptions
+
+
+SAUCE_USERNAME = os.getenv('SAUCE_USERNAME', None)
+SAUCE_ACCCESS_KEY = os.getenv('SAUCE_ACCESS_KEY', None)
+
+us_ondemand = 'ondemand.us-west-1.saucelabs.com'
+eu_ondemand = 'ondemand.eu-central-1.saucelabs.com'
+
+US_SAUCE_DC_URL = 'https://{}:{}@{}/wd/hub'.format(SAUCE_USERNAME, SAUCE_ACCCESS_KEY, us_ondemand)
+EU_SAUCE_DC_URL = 'https://{}:{}@{}/wd/hub'.format(SAUCE_USERNAME, SAUCE_ACCCESS_KEY, eu_ondemand)
+
+
+class SauceSession():
+
+    def __init__(self, data_center='us', username=None, access_key=None, options=None):
+
+        # TODO: flesh this out
```

Comment:
We should flesh this out in terms of a) figuring out what the options are, and b) making sure we're unified across the languages in how we implement them.